lua/wikis/commons/ExternalMediaList.lua

```diff
@@ -334,13 +336,13 @@ end
 ---Displays the subject's team for a given External Media Link
 ---@param subject string
 ---@param date string
----@return string?
+---@return Widget?
 function MediaList._displayTeam(subject, date)
 	local _, team = PlayerExt.syncTeam(subject, nil, {date = date})
 	if not team then
 		return
 	end
-	return Team.icon(nil, team, date)
+	return TeamInline{name = team, date = date, displayType = 'icon'}
```

Comment:
Maybe use `OpponentDisplay.InlineOpponent` or `OpponentDisplay.InlineTeamContainer` for these?

I think going through opponent display is probably better and only using TeamInline as private?

Reply:
I mean in general, not just in EML